lib/Doctrine/Record/Filter/Standard.php

```diff
@@ -34,21 +34,15 @@
 class Doctrine_Record_Filter_Standard extends Doctrine_Record_Filter
 {
     /**
-     * filterSet
-     * defines an implementation for filtering the set() method of Doctrine_Record
-     *
-     * @param mixed $name                       name of the property or related component
```

Comment:
We should keep the phpdoc here, and define it as `string` (as aliases are string only)
wdyt?